src/app/app-routing.module.ts

```diff
@@ -4,15 +4,18 @@ import { RegistrationFormComponent } from './components/login-registration/regis
 import { FeedComponent } from './components//authorized/feed/feed.component';
 import { LoginFormComponent } from './components/login-form/login-form.component';
 import { SubredditsComponent } from './components/authorized/subreddits/subreddits.component';
+import { SubredditComponent } from './components/authorized/subreddit/subreddit.component';
 import { AuthGuard } from './guards/auth.guard';
 import { WelcomePageComponent } from './components/welcome-page/welcome-page.component';
 
 const routes: Routes = [
+    { path: 'feed', component: FeedComponent },
+    { path: 'registration', component: RegistrationFormComponent },
+    { path: 'createSubreddit', component: SubredditComponent },
```

Comment:
Paths should be in lowercase letters and use "-" if needed.

Reply:
corrected